_includes/1kcontributor-list-band.html

```diff
@@ -0,0 +1,32 @@
+<div class="full-width-bg component">
+  <div class="grid-wrapper">
+    <div class="width-12-12 width-12-12-m">
+      <h2 class="mt-0">Quarkus is 1000% for the community </h2>
+    </div>
+    <div class="width-6-12 width-12-12-m">
+      <p>Quarkus has reached a significant milestone by amassing a community of over 1,000 contributors who have propelled the project forward since its inception in June 2018. The Community has played a vital role in the development and growth of Quarkus. Through their collective efforts, these contributors have been instrumental in adding new features, identifying and resolving bugs, and rigorously testing the platform to ensure its stability and reliability. The diversity of the Quarkus community, which includes developers, engineers, and enthusiasts from around the world, has been a key factor in its success, bringing a wealth of expertise and innovative ideas to the table. The remarkable achievement of reaching 1,000 contributors is a testament to the project's growing popularity, the quality of its codebase, and the unwavering commitment of the individuals who have dedicated their time and skills to making Quarkus a true standout in the open-source software landscape. The Quarkus leadership team would like to say thank you to everyone who've contributed to this incredible community.</p>
+    </div>
+    <div class="width-6-12 width-12-12-m">
+      <img src="{{site.baseurl}}/assets/images/1kcontributors/1kcontributors_image.png">
+    </div>
+  </div>
+  <div class="width-12-12 width-12-12-m ">
+    <h2>The Amazing 1000 Contributors</h2>
+    <p>We'd like to recognize and thank these 1000 contributors who've dedicated their time and efforts to take Quarkus from just an idea to the one of the best Java frameworks in open source.</p>
+</div>
+  <div class="grid-wrapper contrib-cards">
+  {% for item in site.data.1kcontributor-list.links %}
+      <div class="card">
+        <a href="{{ item.url }}" target="_blank"></a>
+        <div>
+          <img class="headshot" src="{{ item.avatar }}">
+        </div>
+        <div>
+          <p class="title">{{ item.name }}</p>
+          <p class="description">Github: {{ item.login }}</p>
```

Comment:
```suggestion
          <p class="description">{{ item.login }}</p>
```
 I would just remove it - having github printed 1000 times doesn't provide  much info:)